save-demo/src/main/kotlin/com/saveourtool/save/demo/utils/HttpUtils.kt

```diff
@@ -2,20 +2,26 @@
 
 package com.saveourtool.save.demo.utils
 
-import com.saveourtool.save.core.logging.logInfo
+import com.saveourtool.save.core.logging.describe
 import io.fabric8.kubernetes.api.model.Pod
 import io.fabric8.kubernetes.client.KubernetesClient
 import io.ktor.http.*
 import io.ktor.utils.io.*
 import io.ktor.utils.io.jvm.javaio.*
-import kotlinx.coroutines.delay
+import okhttp3.internal.closeQuietly
+import org.slf4j.LoggerFactory
 import org.springframework.core.io.buffer.DataBufferUtils
 import org.springframework.core.io.buffer.DefaultDataBufferFactory
 import reactor.core.publisher.Flux
+import java.io.IOException
 import java.nio.ByteBuffer
 import kotlin.random.Random
 
 private const val DEFAULT_BUFFER_SIZE = 4096
+private const val PORT_RANGE_FROM = 20000
+private const val PORT_RANGE_TO = 30000
+
+private val logger = LoggerFactory.getLogger("c.s.s.d.u.HttpUtils")
```

Comment:
why such name? logger prints `c.s.s.d.u.` because it's configured in such way. it should be proper class name

Reply:
It is done in order to provide better information about the file that prints logs